install_scripts/setup_sadm.sh

```diff
@@ -12,6 +12,27 @@ Server = https://repo.prologin.org/
 EOF
 fi
 
+sed -i 's/^[#]IgnorePkg.*$/# Ignoring all of the following packages\
+# Install utilities\
+IgnorePkg = arch-install-scripts syslinux mkinitcpio-nfs-utils\
+# Boot and file systems utilities\
+IgnorePkg = nbd tftp-hpa conntrack-tools openresty ipxe-sadm-git nfs-utils\
+# Network\
+IgnorePkg = openssh dhcp bind ipset dnsutils rsync tcpdump\
```

Comment:
Same, why would we have to pin these

Reply:
Fixed according to what was said in #120